RustCrypto/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+authors = ["Brian Smith <brian@briansmith.org>"]
+name = "crypto_bench_rustcrypto"
+version = "0.1.0"
+
+[lib]
+name = "crypto_bench_rustcrypto"
+path = "RustCrypto.rs"
+
+[dependencies.crypto_bench]
+path = "../crypto_bench"
+
+[dependencies]
+#sha2 = { git = "https://github.com/RustCrypto/hashes" }
+sha2 = { git = "https://github.com/RustCrypto/hashes", features = ["asm"] }
```

Comment:
Let's just use the latest release on crates.io, like we now do for the other crates.

Probably it makes sense for the benchmarks to have an `asm` feature that, when provided, enables the `asm` feature of this crate, and otherwise doesn't enable it. This should be documented in the README.

Reply:
Done